### DISCUSSION TITLE
Is there a way to avoid fade in fade out on jingles

### DISCUSSION THREAD

**Discussion by gmantzak** [AzuraCast/AzuraCast#5836] [Q&A] [answered]
Hi All,

First of all, I would like to congratulate all the people who got involved in the thinking and developing process of this fantastic platform.
I have a radio station that is currently running on a commercial platform which am about to migrate in the cloud with Azuracast.
I am using AzuraCast v0.17.4
I have a problem with fade in/out of jingles. Is there a way to configure the application to avoid applying fading on them?

Maybe linked to this? https://github.com/savonet/liquidsoap/issues/1086

Thank you very much
Br
G.

**Comment by paulcjkp** [comment, marked as answer]
When editing tags, you can go to Advanced and change the Custom Fading (set to 0).  However, there does seem to be some issues with Liquidsoap ignoring these randomly (seems to be due to track length).  See more here: https://github.com/savonet/liquidsoap/issues/2611

**Reply by gmantzak** [reply]
Thank you very much @paulcjkp !!
I changed the config in the advanced settings for each jingle to monitor it a bit. I will follow up also the thread in Liquidsoap
